ci/pipeline.yml

```diff
@@ -101,7 +93,7 @@ jobs:
       - postfix-config/bosh/opsfiles/production.yml
       vars_files:
       - terraform-yaml/state.yml
-      - common/secrets.yml
+      - common-production/production-postfix-decrypted.yml
```

Comment:
We'll want to rename these without the `-decrypted` for the sake of convention.

Reply:
Hmm.  This was what Steve did, and I like it because we can roll forward and back without sadness.  But you are leading the charge, so I'll make it so.

Reply:
Yeah I'm impartial really, alas, the convention has been set.